packages/slice-machine/lib/models/common/Environment.ts

```diff
@@ -11,8 +12,7 @@ export interface BackendEnvironment {
   manifest: Models.Manifest;
   repo: string;
   changelog: PackageChangelog;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  mockConfig: any;
+  mockConfig: CustomTypeMockConfig;
```

Comment:
I like that it may looks like it's type but I looked at the type it's a `Record<string, any>` 😂 .
This will be improved by Arnaud's PR i believe.

Reply:
I think that's a `GlobalMockConfig`, but it doesn't change much :)